### PR TITLE
installation: pin invenio-celery <5

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

**Comment by fenekku** [comment]
I guess I will close the one I made yesterday https://github.com/inveniosoftware/invenio-celery/pull/69

**Comment by ntarocco** [comment]
@lnielsen unfortunately Travis if failing because of a version conflict. And also, why removing `isort` checks?

File: setup.py [hunk 1]
```diff
@@ -16,19 +16,19 @@
...
+    "pytest-isort>=1.2.0",
+    "pytest-pycodestyle>=2.2.0",
+    "pytest-pydocstyle>=2.2.0",
+    "pytest>=6,<7",
 ]
 
 extras_require = {
     'docs': [
-        "Sphinx>=1.4.2",
+        "Sphinx>=3",
```
Comment (ppanero) [inline setup.py:31]: Is this something we should start to integrate systematically? many modules have `< 3`

Comment (lnielsen) [reply setup.py:31]: yes

**Comment by lnielsen** [comment]
> @lnielsen unfortunately Travis if failing because of a version conflict. And also, why removing `isort` checks?

I'm not removing isort, but instead moving them into pytest as part of the check (the ``--isort`` in pytest.ini).

**Pull request merged**